hw/top_earlgrey/ip/ast/data/ast.hjson

```diff
@@ -12,6 +12,7 @@
   bus_interfaces: [
     { protocol: "tlul", direction: "device" }
   ],
+  no_auto_alert_regs: "True",
```

Comment:
Is there a reason why this switch was needed?
I believe AST does not specify alerts in the Hjson, right?

Reply:
Yes, DV asked for it.

Reply:
ok, let's leave it there then.